lib/core/output_sink/base_sink.rb

```diff
@@ -18,7 +18,7 @@ def ingest_multiple(_documents)
       end
 
       def delete(_id)
-        raise 'not_implemented'
+        raise 'not implemented'
```

Comment:
Should this just use a [NotImplementedError](https://ruby-doc.org/core-2.5.0/NotImplementedError.html)?

Reply:
TIL, but this is worth a separate PR as we have similar cases in other places.